scripts/sponsoring/README.md

```diff
@@ -0,0 +1,15 @@
+# Add sponsoring functions to proxy
```

Comment:
Do we want this file here or in the root readme ?

Reply:
Info in this readme are related to the diff upgrade related to current deployed contracts, if someone wants to deploy poco somewhere else these new scripts/readme should be useless
Tell me what you think, anyway I can move the content somewhere else if you have an idea :ear: ?

Reply:
You are right :) maybe put the link to this file in the changelog ?